src/components/WrapperFavorites.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+import Title from './Title';
+import Container from './Container';
+import Slider from './Slider';
+import SmallInfoCard from './SmallInfoCard';
+
+import api from 'src/service/api';
```

Comment:
Please use relative import, as done in other places (other imports) => import api from '../service/api'